MultiTable/lens.json

```diff
@@ -37,6 +37,7 @@
     "1.0.19": "Aspect name case insensitivity - missed one",
     "1.0.20": "Add google analytics tracking for search, onclick modals and links, and toggle checkbox",
     "1.0.21": "Prevent reverse tabnabbing",
-    "1.0.22": "Add RealtimeChangeHandler arg validation and tests"
+    "1.0.22": "Add RealtimeChangeHandler arg validation and tests",
+    "1.0.23": "Update onSampleUpdate to add a sample when not found"
```

Comment:
You also need to update the version up above.

Reply:
I do not understand what you are trying to say.

Reply:
line 7 of this file.

Reply:
Got it.

Reply:
maybe describe like this: "Update the sample update handler to add a sample if it's not already being tracked" ?